### DISCUSSION TITLE
How to import security info for big query via cli?

### DISCUSSION THREAD

**Discussion by zmitry** [apache/superset#20595] [Q&A / Help]
Hi, I'm trying to automate superset deployment using terraform. I'm using the helm chart managed by superset. I want to provision databases, databases(big query in my case) and datasets but I can't provision security info that goes into "encrypted_extra".
this is my datasources file. How can I provide security info to superset without using UI https://superset.apache.org/docs/databases/bigquery/#connecting-to-bigquery ?
```
  import_datasources.yaml: |
    databases:
      - database_name: Google BigQuery
        sqlalchemy_uri: bigquery://
        cache_timeout: null
        expose_in_sqllab: true
        allow_run_async: true
        allow_ctas: yes
        allow_cvas: false
        allow_file_upload: false
        uuid: 3429cbdc-0d6f-40b3-be3c-07e10364a0a7
        version: 1.0.0
        extra: |
            {
            "allows_virtual_table_explore": true,
            "cost_estimate_enabled": false,
            "metadata_params": {},
            "engine_params": {} }
 ```

**Comment by zmitry** [comment]
I've ended up using workload identity so I don't need any creds but it won't work for non gcp setup

**Reply by rusackas** [reply]
There are some folks asking how to do this on [slack](http://bit.ly/join-superset-slack) if you don't mind providing more details.

**Reply by akhileshmalavalli** [reply]
@zmitry Is there any doc on how to set this up?
